ha/core/controllers/cluster_controller.py

```diff
@@ -28,6 +28,14 @@ def __init__(self):
         """
         super(ClusterController, self).__init__()
 
+    @controller_error_handler
```

Comment:
For all class no need to add 
`@controller_error_handler`
It is only used if function is exposed

Reply:
Yes, Not needed. I will remove it.